controllers/k8ssandra/reaper.go

```diff
@@ -290,3 +314,69 @@ func (r *K8ssandraClusterReconciler) addClusterToExternalReaper(
 	}
 	return result.Continue()
 }
+
+func addManagementApiSecretsToReaper(
+	ctx context.Context,
+	remoteClient client.Client,
+	kc *api.K8ssandraCluster,
+	actualDc *cassdcapi.CassandraDatacenter,
+	logger logr.Logger,
+	authConfig *cassdcapi.ManagementApiAuthManualConfig,
```

Comment:
Suggestion: could you get this from the kc object to avoid the additional parameter? This function has a lot already.

Reply:
I'm sorry but I don't think removing the `authConfig` function parameter leads to simpler `addManagementApiSecretsToReaper` function. 
The reason is that the `authConfig` passed into this function comes from the `usesHttpAuth` function, which picks it from one of the two possible palces in the `K8ssandraCluster` spec. I'd have to

Reply:
OK, closing as this was only a suggestion